components/progresscircle/stories/progresscircle.stories.js

```diff
@@ -0,0 +1,55 @@
+// Import the component markup template
+import { Template } from "./template";
+
+export default {
+  title: "Progresscircle",
+  description: "Progress circles show the progression of a system operation such as downloading, uploading, processing, etc. in a visual way. They can represent determinate or indeterminate progress.",
+  component: "Progresscircle",
+  argTypes: {
+    size: {
+      name: "Size",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["small", "medium", "large"],
```

Comment:
Typically at this level, size is t-shirt sizing. `Small`, `medium`, and `large` here seem more aligned with platform scale. Is that accurate?  If so, you can call the platform scale values from the template inside the `...globals` spread property (it's defined in the parameters file at the storybook level.

Reply:
> Typically at this level, size is t-shirt sizing.

Yeah, that makes sense.

> Small, medium, and large here seem more aligned with platform scale. Is that accurate?

It's interesting: I think these are more akin to t-shirt sizes, but without an extra-large size. I was double-checking against the [design spec](https://spectrum.adobe.com/page/progress-circle/#Design-checklist) (see "all platform scales"). What do you think? Leave as is?

Reply:
Oh interesting yeah - I think you're right. Should we use `s`, `m`, `l` to align with the other size values in storybook?

Reply:
Yeah, that's good for consistency's sake. I'll make that change.